dev_guide/builds.adoc

```diff
@@ -519,9 +531,42 @@ For example, defining a custom HTTP proxy to be used during build:
 ----
 ====
 
+[[source-code]]
+
+== Git Repository Source Options
```

Comment:
i'd prefer to see this nested under Build Sources

which would contain:
Git Source
Docker Source
Binary Source (in the near future)

it's a bit odd having Git Repository Source Options as a top level heading.